src/vtquery.cpp

```diff
@@ -324,12 +342,12 @@ struct Worker : Nan::AsyncWorker {
 
             // create properties object
             v8::Local<v8::Object> properties_obj = Nan::New<v8::Object>();
-            v8::Local<v8::Object> tilequery_properties_obj = Nan::New<v8::Object>();
             for (auto const& prop : feature->properties) {
-                properties_obj->Set(Nan::New<v8::String>(prop.first).ToLocalChecked(), get_property_value(prop.second));
+                set_property(prop, properties_obj);
```

Comment:
@flippmoke this loop never ... _loops_. Am I referencing `feature->properties` correctly?

Reply:
Fixed!